fsevents.js

```diff
@@ -6,6 +6,13 @@
 /* jshint node:true */
 'use strict';
 
+if (process.platform !== 'darwin') {
+  var err = new Error('Cannot find module \'fsevents\'');
```

Comment:
Why replicate `require`'s error? Couldn't we just write our own, more accurate, custom error message?

```js
'fsevents module is not compatible with ' + process.platform
```

I could imagine your latest suggestion still being confusing to people. They never tried to require `fsevents-win32`, and what happens when they go looking for it or attempting to install it?

I don't think there's much chance of any users encountering this error message anyway, though. Module authors incorporating `fsevents` will hopefully be savvy enough to set it up properly.